modules/affile/affile-grammar.ym

```diff
@@ -269,6 +271,7 @@ dest_afpipe_option
 	: dest_writer_option
 	| dest_driver_option
 	| file_perm_option
+	| KW_TIME_REAP '(' nonnegative_integer ')'		{ affile_dd_set_time_reap(last_driver, $3); }
 	;
 
```

Comment:
I don't think it its applicable to pipe()

Reply:
The PR was inspired by a mailing list question. Someone had problems using `pipe()`, because `time-reap()` closed the pipe.

I think, in that case, we should just disable the time-reap functionality in case of named pipes.
`time-reap(0)` IMHO can be useful in other places: non-templated file destinations, other file-based internal modules.

Reply:
I did not want to remove existing functionality (time-reap for pipe). I am totally okay with removing it, though, that would solve the user's issue, too. Up to you.